content/en/css.md

```diff
@@ -74,20 +78,24 @@ CSS is most efficient when its selectors are [extremely specific with limited DO
 
 The descendant selector (the space character) is the most expensive selector in CSS. The child selector (the &quot;`>`&quot; character) is also expensive, especially when the rules are tag names rather than classes or IDs. Avoid both. Try applying a class to the element you want to target instead.
 
-```css
+```
 /* BAD */
 button#back-button { ... }
 .popular ul li a { ... }
 .popular > ul > li > a { ... }
 
 /* GOOD */
-#back-button { ... }
+.back-button { ... }
 .popular-link { ... }
 .popular-link { ... }
 ```
 
 Avoid using the `!important` keyword. Treat it like the nuclear option, only to be used in the most extreme of cases. There is usually another way to achieve the same goal without causing headaches for developers in the future who are either trying to debug a styling issue or trying to use normal specificity to override a style for a particular element only to find that they can't.
 
+While the ID selector is the most efficient selector, it's use should be minimized as it's highly specific nature makes it difficult to override. It is recommended to restrict selectors to purely class names, preferrably selectors that utilize only one class name so that all selectors have a comparable level of specificity. 
```

Comment:
~some grammar stuff~
While the ID selector is the most efficient selector, ***its*** use should be minimized as ***its*** highly specific nature makes it difficult to override. It is recommended to restrict selectors to purely class names, ***preferably*** selectors that utilize only one class name so that all selectors have a comparable level of specificity.

~some possible content to use instead~
Use the lowest level of specificity necessary to get the desired results. This means use of the ID selector should be minimized. Oftentimes creating a new class is preferable to using inheritance or specificity to target an element or elements.

.tree-cell.step-two {  } <--- okay but maybe too specific
.step-two {   } <-- normally just do this

.tree-cell-step-two {   } <--- good
.tree-cell .step-two {   } <-- bad performance?